### DISCUSSION TITLE
第三者レビュー ログ-ADR

### DISCUSSION THREAD

**Discussion by creature-water-valley** [ws-4020/mobile-app-crib-notes#568] [General]
#### ログ出力の方針

- コンテキスト
  - コンテキストの時点でログに含める内容は決まっているかのように見えたのが少し気になりましたので、2階層目の中点部分は書かなくてもいいように思いました
- ログ出力に必要な機能
  - Typoと思われる `FIrebase`がありました
- サードパーティー・ライブラリの使用
  - 「自作した場合のコストとモジュール管理の煩雑さを比較検討し、採用しない」のところは比較過程が見えないので結論までの道がよくわからなかったです。自作する方がなんでコスト低いのか、npm moduleは他にも使ってると思うがログ系は特に煩雑なのか、など。
  
#### ログ出力の用途

- ログ出力の用途
  - Typoと思われる 考えらます がありました

**Comment by creature-water-valley** [comment]
### 対応内容

- コンテキストのログに含める内容の２階層目は削除する
- FIrebase -> Firebase
- 「特定のログライブラリに依存せず、現場によってログライブラリの実装を切り替えることができるように」という理由づけに変更
- 考えらます -> 考えられます

**Comment by kozake** [comment]
### 対応PR

https://github.com/ws-4020/mobile-app-crib-notes/pull/563

**Comment by creature-water-valley** [comment]
対応済みなのでクローズ
